tests/plugins/tableselection/manual/safaritheadlastcolumn.html

```diff
@@ -0,0 +1,61 @@
+<p><button id="select">Select the last column</button></p>
+
+<div id="editor">
+	<table border="1">
+		<thead>
+			<tr>
+				<th scope="col">Column 1</th>
+				<th scope="col">Column 2</th>
+				<th scope="col">Column 3</th>
+				<th scope="col">Column 4</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<th scope="row">Cell 1.1</th>
+				<td>Cell 1.2</td>
+				<td>Cell 1.3</td>
+				<td>Cell 1.4</td>
+			</tr>
+			<tr>
+				<th scope="row">Cell 2.1</th>
+				<td>Cell 2.2</td>
+				<td>Cell 2.3</td>
+				<td>Cell 2.4</td>
+			</tr>
+			<tr>
+				<th scope="row">Cell 3.1</th>
+				<td>Cell 3.2</td>
+				<td>Cell 3.3</td>
+				<td>Cell 3.4</td>
+			</tr>
+			<tr>
+				<th scope="row">Cell 4.1</th>
+				<td>Cell 4.2</td>
+				<td>Cell 4.3</td>
+				<td>Cell 4.4</td>
+			</tr>
+			<tr>
+				<th scope="row">Cell 5.1</th>
+				<td>Cell 5.2</td>
+				<td>Cell 5.3</td>
+				<td>Cell 5.4</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
+
+<script src="../_helpers/tableselection.js"></script>
+<script>
+	if ( !CKEDITOR.env.safari ) {
```

Comment:
As there is a part of test that requires mouse, we should either ignore the whole test on mobile or add a note that the second point shouldn't be executed there.